### PR TITLE
feat(Skip To Content): Added Skip To Content Component and Page breadcrumb styling

### PR DESCRIPTION
**Pull request by dlabaj**
Created the skip to content component for react.

#1496, #1733 

**Additional issues**:

None

### CODE REVIEW

**Comment by mcoker** [comment]
Looks good, but can you add `<a id="main-content"></a>` before the first page section? You can see that link in any of the demos here except for the first one https://pf4.patternfly.org/demos/Page/examples/

I've opened an issue to add that link to the first demo, and the skip-to-content component to some of our other demos that use the page component https://github.com/patternfly/patternfly-next/pull/1680

File: packages/patternfly-4/react-core/src/components/Page/Page.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ const propTypes = {
   header: PropTypes.node,
   /** Sidebar component for a side nav (e.g. <PageSidebar />) */
   sidebar: PropTypes.node,
+  /** Skip to content componet for the page */
```
Comment (kmcfaul) [inline packages/patternfly-4/react-core/src/components/Page/Page.js]: Small spelling fix just for the generated docs. Everything else looks great!

**Comment by mcoker** [comment]
for the `<a id="#main-content-page-layout-default-nav>` elements, you need to remove the `#` from the `id` attribute. That should only exist in the `href` on the link.

So it's `<a class="pf-c-skip-to-content ..." href="#some-id">` and the anchor in the page is `<a id="some-id">`

Also the anchor in the main content should come before all content, but after any sort of navigation. The idea is that if a non-sighted person is navigating your website, and they're just going from page to page, we want there to be a link as the first thing on the page that lets them skip the header and any navigation, and dive right to the start of the page content. In these examples, the anchor comes after the first block of content, so we should fix that. Is it up to users to put their own anchor on the page, or do we do it for them?

File: packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: 'Skip To Content'
+section: 'components'
+cssPrefix: 'pf-c-skip-to-content'
+---
+
+## Simple SkipToContent
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.md]: can we make this sentence case.  `skip to content`

File: packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/SkipToContent/skip-to-content.css';
+import buttonStyles from '@patternfly/patternfly/components/Button/button.css';
+import { css, getModifier } from '@patternfly/react-styles';
+
+export interface SkipToContentProps
+  extends React.DetailedHTMLProps<React.AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement> {
+  /** Sets the base component to render. Defaults to an anchor */
+  component: any,
+    /** The skip to content link. */
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.tsx]: nit. remove extra space.

File: packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Skip To Content'
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md]: sentence case here

File: packages/patternfly-4/react-core/src/components/Page/Page.d.ts [hunk 5]
```diff
@@ -5,6 +5,7 @@ export interface PageProps extends HTMLProps<HTMLDivElement> {
   className?: string;
   header?: ReactNode;
   sidebar?: ReactNode;
+  skipToContent?: ReactNode,
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/src/components/Page/Page.d.ts]: is breadcrumb missing here?

Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/Page/Page.d.ts]: yep good catch.

File: packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md (hunk 4 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md]: Done.

File: packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.tsx (hunk 3 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.tsx]: Done.

File: packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.md (hunk 2 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/SkipToContent/SkipToContent.md]: Done.

File: packages/patternfly-4/react-core/src/components/Page/Page.d.ts [hunk 6]
```diff
@@ -5,8 +5,10 @@ export interface PageProps extends HTMLProps<HTMLDivElement> {
...
   isManagedSidebar?: boolean;
   onPageResize?: Function;
+  breadcrumb: ReactNode;
```
Comment (dgutride) [inline packages/patternfly-4/react-core/src/components/Page/Page.d.ts]: Should be marked optional

File: packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md (hunk 4 above)
Comment (dgutride) [reply packages/patternfly-4/react-core/src/components/SkipToContent/examples/SimpleSkipToContent.md]: still waiting . . .

File: packages/patternfly-4/react-core/src/components/Page/Page.d.ts (hunk 6 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/Page/Page.d.ts]: done.

**Comment by dlabaj** [comment]
@dgutride Done updated to Simple skip to content to be the correct title for the md file.

**Pull request merged**